zinc/src/test/scala/sbt/inc/cached/CommonCachedCompilation.scala

```diff
@@ -86,6 +111,7 @@ abstract class CommonCachedCompilation(name: String)
 
     val prefix = tempDir.toPath.toString
 
+    // TODO(jvican): Add files that are missing...
```

Comment:
This is just a comment to check not only the stamps, but also the files. Will fix in next commit. Good catch, forgot about this.